### PR TITLE
Add class name argument to `ic`

### PR DESCRIPTION
**Pull request by mrmacete**
tests here: https://github.com/radare/radare2-regressions/pull/828

### CODE REVIEW

File: libr/core/cmd_info.c [hunk 1]
```diff
@@ -481,18 +481,29 @@ static int cmd_info(void *data, const char *input) {
...
 				if (obj) {
 					if (input[2]) {
+						int idx = -1;
+						const char * cls_name = NULL;
+						if (r_num_is_valid_input (core->num, input + 2)) {
+							idx = r_num_math (core->num, input + 2);
+						} else {
+							const char * first_char = input + ((input[1] == ' ') ? 1 : 2);
+							int not_space = strspn (first_char, " ");
+							if (*(first_char + not_space)) {
```
Comment (radare) [inline libr/core/cmd_info.c]: first_char[not_space] seems more readable to me

File: libr/core/cmd_info.c [hunk 2]
```diff
@@ -481,18 +481,29 @@ static int cmd_info(void *data, const char *input) {
...
-				int count = 0;
 				if (obj) {
 					if (input[2]) {
+						int idx = -1;
+						const char * cls_name = NULL;
+						if (r_num_is_valid_input (core->num, input + 2)) {
+							idx = r_num_math (core->num, input + 2);
+						} else {
+							const char * first_char = input + ((input[1] == ' ') ? 1 : 2);
+							int not_space = strspn (first_char, " ");
```
Comment (radare) [inline libr/core/cmd_info.c]: uhm, not sure how portable is `strspn`. its from C90, and seems to build fine with mingw, so let's keep it, but its the first time this is used in r2 iirc. :)

Comment (mrmacete) [inline libr/core/cmd_info.c]: oops, well... i can do a while either

Comment (radare) [inline libr/core/cmd_info.c]: nah its ok :) if it works we may probably update other parts of the codebase. it's not really a complex thing

**Pull request merged**